### PR TITLE
fix: delete cars

### PR DESCRIPTION
**Pull request by abdou059**

### CODE REVIEW

File: server/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
       "module": "commonjs",
       "esModuleInterop": true,
-      "target": "es6",
+      "target": "es2017",
```
Comment (LinaYahya) [inline server/tsconfig.json]: Why is that?

File: server/src/routes/cars.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 import ExpressWrapper from './ExpressWrapper';
-import { getFilteredCars, getCarsById } from '../controllers';
+import { getFilteredCars, getCarsById, deleteCarsById } from '../controllers';
 
 const carsRouter = Router();
 
 carsRouter.get('/cars', ExpressWrapper(getFilteredCars));
 carsRouter.get('/cars/:id', ExpressWrapper(getCarsById));
-
+carsRouter.delete('/cars/:id', ExpressWrapper(deleteCarsById));
```
Comment (LinaYahya) [inline server/src/routes/cars.ts]: this route is protected, only admin can delete the car

File: server/src/services/cars/deleteCar.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { Car } from '../../db/models';
+
+const deleteCars = async (id) => {
+  const car = await Car.destroy({
+    where: ({
+      id,
```
Comment (LinaYahya) [inline server/src/services/cars/deleteCar.ts:6]: What if id isn't a number, what if there's no car with this id

File: server/src/controllers/cars/deleteCarsById.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { Request } from 'express';
+import { deleteCars } from '../../services';
+
+const deleteCarsById = async (req:Request) => {
+  const { id } = req.params;
+  const del = await deleteCars(id);
```
Comment (LinaYahya) [inline server/src/controllers/cars/deleteCarsById.ts]: no need to return anything

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by mai-mohammed — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by husamkamal — APPROVED** [review]

**Pull request merged**